README.markdown

```diff
@@ -7795,10 +7797,46 @@ tcpsock:setoption
 
 This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality will be implemented in future.
 
+`option` is a string with the option name, and value depends on the option being set:
+
+* `keepalive`
+	Setting this option to true enables sending of keep-alive messages on
+	connection-oriented sockets.
+
+* `reuseaddr`
+	Enable this option indicates that the rules used in validating addresses
+	supplied in a call to bind should allow reuse of local addresses.
+
+* `tcp-nodelay`
+	Setting this option to true disables the Nagle's algorithm for the connection.
+
+* `sndbuf`
+	Sets the maximum socket send buffer in bytes. The kernel doubles this value
+	(to allow space for bookkeeping overhead) when it is set using setsockopt().
+
+* `rcvbuf`
+	Sets the maximum socket receive buffer in bytes. The kernel doubles this value
+	(to allow space for bookkeeping overhead) when it is set using setsockopt().
+
+These options described above are supported in `vXXX`, and more options will be implemented in future.
+
```

Comment:
these new added line better move behind the line 7823:
https://github.com/openresty/lua-nginx-module/pull/1790/files#diff-0a369498a5a8db3ac8fa606b544c9810R7823

Reply:
accept